docs/milestone-2-steps.md

```diff
@@ -0,0 +1,183 @@
+# Deployment Instructions
+
+## Limitations
+
+- Bucket versioning not supported: [S3C-4456](https://scality.atlassian.net/browse/S3C-4456)
+- Buckets and objects not created via VMware OSE do not appear in vCloud Director: [S3C-4199](https://scality.atlassian.net/browse/S3C-4199)
+
+## Prerequisite
```

Comment:
```suggestion
## Prerequisites
```

Reply:
Changed in squash commit: [`bc337ec` (#52)](https://github.com/scality/vmware-ose-scality/pull/52/commits/bc337ec87d8758d0be6cdf87ef99dd6f005236dc)